monty/exts/api/wikipedia.py

```diff
@@ -73,7 +72,7 @@ async def wikipedia_search_command(self, ctx: commands.Context, *, search: str)
         if contents:
             embed = disnake.Embed(title="Wikipedia Search Results", colour=disnake.Color.blurple())
             embed.set_thumbnail(url=WIKI_THUMBNAIL)
-            embed.timestamp = datetime.utcnow()
+            embed.timestamp = disnake.utils.utcnow()
```

Comment:
I'd rather not use disnake.utils.utcnow. I know its there, but I don't want to use a util of a library for this.

Reply:
It's already being used in other places, but I can refactor it to use a new util method?

Reply:
please do

Reply:
https://github.com/onerandomusername/monty-python/pull/240/commits/711927ce8c17aeac7d295b41925ebdb855f5bbbd